lib/far_mar/market.rb

```diff
@@ -0,0 +1,72 @@
+class FarMar::Market < FarMar::Base
+
+attr_accessor :id,
+              :name,
+              :address,
+              :city,
+              :county,
+              :state,
+              :zip
+
+def initialize(attrs)
+  @id       = attrs[:id]
+  @name     = attrs[:name]
+  @address  = attrs[:address]
+  @city     = attrs[:city]
+  @county   = attrs[:county]
+  @state    = attrs[:state]
+  @zip      = attrs[:zip]
+end
+
+  def self.path
+    "/Users/Chris/Code/class_projects/farmar/far_mar/support/markets.csv"
+  end
+
+  def self.all
+    @all ||= all_calc
+  end
+
+  def self.all_calc
+    CSV.read(path).map do |line|    # Read the markets.csv file
+      new(                          # for each row
+        id:      line[0].to_i,      # create a new instance of FarMar::Market
+        name:    line[1],           # create an array of those instances
+        address: line[2],           # return that new array
+        city:    line[3],
+        county:  line[4],
+        state:   line[5],
+        zip:     line[6].to_i
+      )
+    end
+  end
+
+  def vendors
+    vendor_list = FarMar::Vendor.all.group_by do |obj|
```

Comment:
This method should reuse the `FarMar::Vendor.by_market()` method, the result of them is the same.